docs_src/design/ucr/Extending-Device-Data.md

```diff
@@ -0,0 +1,78 @@
+## Extending Device Data
+This UCR describes the Use Case for Extending of Device Data by Application Services for a given south-bound Device.
+
+### Submitters
+- Tom Brennan (Eaton)
+
+## Change Log
+- [pending](https://github.com/edgexfoundry/edgex-docs/pull/845) (2022-09-12)
+
+
+### Market Segments
+Any that deploy EdgeX systems with analytics, utility, or north-bound microservices that add new Device Resources
+that are extensions of the original south-bound or service-based Device data.
+
+### Motivation
+We find a consistent need as we design microservices for our industrial products:
+The new analytics, utility, and north-bound microservices almost always need to add Device Resources to manage their configuration, transforms, and status reporting. These Resources are usually needed on a per-Device basis (rather than just overall service configuration or status), which can be seen as extending (adding to) the data of the original south-bound devices.
+
+Adding configuration and status via Resources that **extend** the original south-bound Device
+make this configuration and status data easily accessible and translatable to other Application Services and to the UI via REST;
+we think that this general solution is better than disparate solutions which add custom APIs in each Application Service to Get and Set this data.
+
+What is needed is a common means of showing the relationship between these added Resources, their owning service,
+and the original south-bound Device Resources; that is, to indicate that these Resources "extend" the original Device data.
+
+It is desirable that the means of conveying this information become standardized for those EdgeX microservices
+which provide and use it, hence proposing here that there be a common EdgeX way defined to do this.
+
+### Target Users
+- Product Developers
+- Device Owner
+- Device User
+- Device Maintainer
+- Cloud User
+- Service Provider
+- Software Integrator
+
+### Description
+Picture the extremely simple case of a south-bound sensor device that just measures Temperature and Humidity and provides these as Device Resources. If we then add analytics and north-bound microservices:
+- A Trending service that needs Device Resources to indicate that Temperature and Humidity are trended for, eg, Minimum, Average, and Maximum over a 1 hour trend interval.
+- An Alarming service that needs Device Resources to describe the Alarm Rules used to monitor Temperature and Humidity, plus a device-level InAlarm status.
+- A Cloud service that reports not just the Temperature and Humidity but also their Trend configuration and Alarm Rule Resources. In addition, the Cloud service adds its own Resources to direct the Cadence with which this Device's data is reported.
+
+Now scale this up to 100 such Temperature/Humidity sensors, and it grows difficult to match all of the free-standing 
```

Comment:
I am having difficulty understanding this sentence.  I get the scale issue and tracking the data back to its originating service.  Perhaps it is the term "free-standing" here that confuses me.  What is a free-standing resource?

Reply:
I was trying to contrast the case of 100 south-bound devices + 200-300 extensions to 300-400 "free standing" devices if the upper layer devices weren't clearly shown as related to their south-bound devices.